### ISSUE TITLE
Newly created projects that are not reviewed by an admin should go from unlisted to listed automatically

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-next#303]
This was an oversight! We want to make sure that if a new project is added and no "admin" reviews the project within 2 weeks then the status automatically gets updated from "unlisted" to "listed".

Projects that are set to be listed (and stay unlisted) should remain unlisted unless an admin reviews them again and changes it.

@mateodaza is this easy to add in?

**Comment by WhyldWanderer** [comment]
Pasting comments from discord.. 

![image](https://user-images.githubusercontent.com/74267818/137512122-a5e84755-5c8d-427f-bb2b-bb5d235581c8.png)
![image](https://user-images.githubusercontent.com/74267818/137512124-d37f4073-442a-4165-9fd7-a98a3924f95e.png)

**Comment by DonnaOftadeh** [comment]
@MoeNick is it related to #377 ? cause 377 automatic un listing for unreviewed projects is now working fine.

**Comment by markoprljic** [comment]
What is the **_final final final ultimate final not gonna change final decision_** on this, please? :)
I'm working on the Create project flow and at the bottom of the page right now I have this disclaimer attached. Let me know if I should remove it or keep it. 

![Image 2021-12-09 at 8 56 53 am](https://user-images.githubusercontent.com/5704285/145356506-13633746-71c9-4c6a-b42e-8ef3ff3d7916.jpg)

**Comment by MoeNick** [comment]
Its implemented now @markoprljic 

Default value after creation will be set to null, means not reviewed. 

In normal scenario, verification team review the project and set it to true or false (listed, delisted) in admin dashboard. 

If it still remained not reviewed after 2 weeks (its configurable)  it will be set to true (listed) automatically.

**Comment by markoprljic** [comment]
I'm not sure I understand the answer :(
Are projects listed after they are created, or are they unlisted by default and waiting to be reviewed?

**Comment by MoeNick** [comment]
Not reviewed and delisted have the same behavior,  they wont show up in the projects page nor in search result. 

We make them different just to tell the user that your project is reviewed and still not listed or still waiting for reviewing. @markoprljic

**Comment by markoprljic** [comment]
> What is the **_final final final ultimate final not gonna change final decision_** on this, please? :) I'm working on the Create project flow and at the bottom of the page right now I have this disclaimer attached. Let me know if I should remove it or keep it.
> 
> ![Image 2021-12-09 at 8 56 53 am](https://user-images.githubusercontent.com/5704285/145356506-13633746-71c9-4c6a-b42e-8ef3ff3d7916.jpg)

Ok so this disclaimer at the bottom can stay?

**Comment by MoeNick** [comment]
Yeah it's clear I think. Just from my side, better we have @WhyldWanderer confirmation.

**Comment by WhyldWanderer** [comment]
YAY! @markoprljic it looks great!

Is it possible for the first bullet point to simply say "Newly published projects will be "unlisted" until reviewed by our team."?
"reviewed by our team" should link to https://docs.giveth.io/dapps/listedUnlisted

The reason is there are more requirements than just being in agreement with the Terms of Service and all of those things are outlined in the guide...

**Comment by markoprljic** [comment]
Done.
